### PR TITLE
PD-37 Add HelpCommand and HelpModal

### PR DESCRIPTION
**Pull request by crapper**

### CODE REVIEW

**Review by Jerrylum — CHANGES_REQUESTED** [review]

File: src/command/Commands.ts [hunk 1]
```diff
@@ -31,6 +32,25 @@ export interface InputSpec<T extends ParameterTypeClass> {
...
 }
 
+export function getCommandUsage(cmd: Command): string {
+  let line = "";
+  if (cmd.name) {
+    line += `${cmd.name}`;
+  }
+  if (!cmd.usage) return line;
+  line += " <";
+  let first = cmd.usage;
```
[the remark this answers is not in the document]
Comment (crapper) [reply src/command/Commands.ts:42]: Updated. Thank you for the reminder.

File: src/command/Commands.ts [hunk 2]
```diff
@@ -31,6 +32,25 @@ export interface InputSpec<T extends ParameterTypeClass> {
   next: InputSpec<ParameterTypeClass> | null;
 }
 
+export function getCommandUsage(cmd: Command): string {
+  let line = "";
+  if (cmd.name) {
```
[the remark this answers is not in the document]
Comment (crapper) [reply src/command/Commands.ts:37]: Removed, there should not be any class without a name, thank you.

File: src/command/Commands.ts [hunk 3]
```diff
@@ -31,6 +32,25 @@ export interface InputSpec<T extends ParameterTypeClass> {
   next: InputSpec<ParameterTypeClass> | null;
 }
 
+export function getCommandUsage(cmd: Command): string {
```
[the remark this answers is not in the document]
Comment (crapper) [reply src/command/Commands.ts:35]: Added, thank you for the reminder!

**Review by Jerrylum — APPROVED** [review]
LGTM

**Pull request merged**